ci/Jenkinsfile-ppc64le-cuda9

```diff
@@ -206,6 +206,16 @@ pipeline {
                        """
                     unstash 'version_info'
                     sh 'echo "Stashed version file:" && ls -l build/'
+                    sh """
+                        echo "Before unstash condapkg:"
+                        ls -l condapkgs  || true
+                        rm -rf condapkgs || true
+                       """
+                    unstash "condapkg"
+                    sh """
+                        echo "After unstash condapkg:"
+                        ls -l condapkgs  || true
+                       """
```

Comment:
I was never able to build conda package for h2o4gpu for ppc64le, hence the corresponding Jenkins files did not have any conda specific code. Are we now able to build it for ppc64le too? That would be awesome step forward for cross arch scoring pipeline deployment